validate/validate.go

```diff
@@ -100,7 +100,9 @@ func (v *Validator) CheckAll() (msgs []string) {
 	msgs = append(msgs, v.CheckMandatoryFields()...)
 	msgs = append(msgs, v.CheckSemVer()...)
 	msgs = append(msgs, v.CheckMounts()...)
-	msgs = append(msgs, v.CheckProcess()...)
+	if v.spec.Process != nil {
+		msgs = append(msgs, v.CheckProcess()...)
```

Comment:
@Mashimiao I think we can do it in this PR. Since the PR does not do anything else.
@q384566678 How about it?